types/slonik/index.d.ts

```diff
@@ -221,15 +221,15 @@ export type QueryResultRowType<ColumnName extends string = string> = {
     [name in ColumnName]: QueryResultRowColumnType;
 };
 
-export type QueryAnyFirstFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<Array<T[keyof T]>>;
-export type QueryAnyFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<T[]>;
-export type QueryFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<QueryResultType<T>>;
-export type QueryManyFirstFunctionType = QueryAnyFirstFunctionType;
+export type QueryAnyFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<ReadonlyArray<T>>;
+export type QueryAnyFirstFunctionType = <T>(...args: QueryMethodParams<T>) => Promise<ReadonlyArray<T>>;
```

Comment:
These aren't readonly arrays, and there's no reason to type them that way. You lose the ability to use utility functions for regular arrays by doing this.